apps/admin-gui/src/app/shared/components/two-entity-attribute-page/two-entity-attribute-page.component.ts

```diff
@@ -128,7 +138,7 @@ export class TwoEntityAttributePageComponent implements OnInit {
         break;
       case 'facility':
         this.facilitiesManagerService.getAssignedUsers(this.firstEntityId).subscribe(users => {
-          this.entityValues = users;
+          this.entityValues = users.sort(this.compareFnUser);
```

Comment:
when we are sorting the names(first name + last name) of members and users I think we really want to sort according to last name(then according to firstname) + count with czech alphabet